timelock-impl/src/main/java/com/palantir/atlasdb/timelock/clock/ClockSkewExceptionLogLimiter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.clock;
+
+@FunctionalInterface
+public interface ClockSkewExceptionLogLimiter {
```

Comment:
nit: we could probably just stick  a `RateLimiter` in the monitor, not sure we need a new interface and impl here

Reply:
good point. I think I stuck this here because I initially thought about doing some testing with a fake ratelimiter, but then later decided it wasn't necessary.